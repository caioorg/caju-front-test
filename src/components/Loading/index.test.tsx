import "@testing-library/jest-dom"
import { render } from "@testing-library/react"
import { LoadingSpinner } from "."

describe("[Component] - Loading", () => {
  it("should render component loading", () => {
    const { asFragment } = render(<LoadingSpinner />)

    expect(asFragment()).toMatchSnapshot()
  })
})
