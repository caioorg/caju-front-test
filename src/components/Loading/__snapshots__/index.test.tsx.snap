// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Component] - Loading should render component loading 1`] = `
<DocumentFragment>
  <div
    class="sc-braxZu hNzEkH"
  />
  <div
    class="sc-gJhJTp fthsbe"
  />
</DocumentFragment>
`;
